### ISSUE TITLE
Cursor behaves as pointer over entire header

### ISSUE THREAD

**Issue by therohitramesh** [hoodiehq/hoodie#815]
The cursor behaves as a pointer over the entire header though the entire header is not a hyperlink.

**Comment by therohitramesh** [comment]
I would like to work on it.

**Comment by gr2m** [comment]
good catch! This is about the header on the http://hood.ie/ website, right?

The source code of the website is rather complicated, sth we desperately want to clean up and make simpler to contribute to. But please give it a go at https://github.com/hoodiehq/hood.ie

**Comment by therohitramesh** [comment]
@gr2m Yes, this is about the http://hood.ie/ website.
I'll look into it and try my best to solve it :)

**Comment by mikethehud** [comment]
Seems that https://github.com/hoodiehq/hoodie-css/pull/53 is a fix but it doesn't touch the scss that causes the problem. I'll submit a pull request shortly.

**Comment by gr2m** [comment]
I honestly don’t know how all the things work together anymore creating a new website is a big project that is way overdue for Hoodie, see https://github.com/hoodiehq/hood.ie/issues/312
